### DISCUSSION TITLE
Automazione luminosità

### DISCUSSION THREAD

**Discussion by ster1um** [indomus/forum#2827] [HUB: Home Assistant]
Ciao a tutti,
volevo creare un'automazione che mi portasse la luminosità di una serie di faretti ad un certo livello se all'accensione sono sotto una determinata luminosità. Questo in quanto se li attivo con una soglia precedentemente impostata bassa alcuni di essi non riescono ad attivarsi.

**Comment by indomus** [comment]
le automazioni non si scrivono così.

https://indomus.it/formazione/esercizi-di-domotica-su-home-assistant-2-definire-diversi-trigger-per-innescare-automazioni/

**Reply by ster1um** [reply]
ho copiato il codice generato dalla creazione guidata di home assistant...

**Reply by indomus** [reply]
già. motivo per cui vi diciamo sempre di non usarla...

**Reply by ster1um** [reply]
sinceramente non capisco la differenza tra quello spiegato nel link e l'automazione generata da HA. 
Poi se vado ad inserire l’automazione come da link genera errori.

**Reply by indomus** [reply]
se non vedi differenze i casi sono due, o abbiamo scritto molto male l'articolo oppure non lo hai letto come dio comanda.

**Reply by ster1um** [reply]
Ti ringrazio, ho risolto rifacendo automazione direttamente da home assistant.

**Reply by KC-inDomus** [reply]
perdendo la chance di imparare a farle a mano mooooolto meglio. peccato

**Reply by ster1um** [reply]
Sono riuscito anche manualmente ma ho preferito rifare tutto via wizard da interfaccia grafica.
Comunque più che INdomus vi dovevate chiamare INtegralista… si scherza ne 🤭🤭🤭

comunque solo a livello informativo, sarebbe possibile creare un automazione che esegua uno schema che ora vi espongo.

es. Luce con luminosità impostata a 10% cambia di stato da spenta ad accesa -> se la luminosità è impostata sotto gli 11% portala al 14% per un 1 sec. e poi riportala alla luminosità iniziale che in questo caso è 10%. 

Follia o fattibile?

**Reply by indomus** [reply]
fattibile. prova con l'editor automazioni di home assistant 👍

**Reply by ster1um** [reply]
Ho provato a smanettare un po’ ma non capisco come richiamare la luminosità iniziale. Non capisco come impostare la variabile che legga la luminosità iniziale e la riporti alla fine.

**Reply by ster1um** [reply]
sono stati cancellati dei messaggi, provo a rispiegare cosa vorrei fare.

Ho una serie di faretti che se vengono accessi con luminosità sotto una certa soglia alcuno fanno fatica ad accendersi. 
quello che volevo fare era un automazione che quanto si accendano con luminosità impostata sotto 14 vadano per 1 sec a 20 e poi ritornino alla luminosità iniziala.
